### ISSUE TITLE
Purpose of the core

### ISSUE THREAD

**Issue by kshedstrom** [ESMG/PyCNAL_core#1]
I believe the purpose of this repo was to hold a grid object to be used in MOM6 tools. Since we only have to worry about the horizontal grid for interpolations, are there grid objects we can use from other packages instead of writing our own? ESMPy and xgcm both have grid objects. What does ours need to have?

**Comment by raphaeldussin** [comment]
I think xgcm has the most comprehensive description of grid objects. It would be a duplication of efforts to write something new. We can use PyCNAL_core as a wrapper to xgcm, which would give the option to switch to a different package in the future, if needed.

**Comment by kshedstrom** [comment]
Do you know about OceanSpy? https://oceanspy.readthedocs.io/en/latest/

**Comment by raphaeldussin** [comment]
yes but I haven't tried it out yet. They use MITgcm mostly but it might work with MOM6 out of the box or with minimal fixes.

**Comment by kshedstrom** [comment]
So far they only support z-coordinates, but there's an effort to support other ocean vertical coordinates. We'll see where it goes...
